### PR TITLE
[stable/grafana] Update documentation to be consistent with the values.yaml file

### PR DESCRIPTION
**Pull request by sarce**
This PR updates the readme documentation to be consistent with the values.yaml file.

### CODE REVIEW

**Comment by cpanato** [comment]
@sarce please bump the chart version

**Comment by sarce** [comment]
@cpanato chart version updated! thanks!

File: stable/grafana/Chart.yaml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 name: grafana
-version: 1.14.6
-appVersion: 5.2.4
+version: 1.14.7
+appVersion: 5.2.5
```
Comment (scottrigby) [inline stable/grafana/Chart.yaml]: Why is the `appVersion` changing? The default `image.tag` is still [5.2.4](https://github.com/helm/charts/blob/master/stable/grafana/values.yaml#L14).

**Review by scottrigby — CHANGES_REQUESTED** [review]
Looks like `appVersion` was accidentally bumped as well.

**Comment by cpanato** [comment]
@scottrigby my bad i did not see that in the review :(

**Comment by sarce** [comment]
Thanks @scottrigby, I've reverted the appversion.

**Pull request merged**